speed-dreams/source-2.2.3/data/data/menu/ResearcherMenu.xml

```diff
@@ -96,6 +96,20 @@
             <attstr name="pushed image" val="data/img/button-left-pushed.png"/>
         </section>
 
+        <!-- Although this is a label, this is not a static control as we want to change it-->
+        <section name="NoBlackBoxError">
+            <attstr name="type" val="label"/>
+            <attnum name="x" val="20"/>
+            <attnum name="y" val="205"/>
+            <!-- "(Hint) " + 37 Spaces so maxlength is set correctly-->
+            <attstr name="text" val="(Hint)                                      "/>
```

Comment:
Having (Hint) in the label is unnecessary, should be empty normally and show an error if one occurs.
```suggestion
            <!-- "Label for showing error messages to the user, needs to be at least 50 characaters long -->
            <attstr name="text" val="                                                                                      "/>
```